docker/sorry-cypress/sorry-cypress.yml

```diff
@@ -0,0 +1,117 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Sorry Cypress Setup
+# ----------------------------------------------------------------------------------------------------------------------
+# this is the docker compose file for the sorry cypress setup
+# ----------------------------------------------------------------------------------------------------------------------
+
+services:
+  mongo:
+    image: mongo:4.4
+    container_name: Mongo
+    hostname: mongo
+    restart: always
+    volumes:
+      - files/mongo:/data/db
+    ports:
+      - 27017:27017
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: 'sorry-cypress'
+      MONGO_INITDB_ROOT_PASSWORD: 'sorry-cypress'
+
+  director:
+    image: agoldis/sorry-cypress-director
+    container_name: Director
+    hostname: director
+    restart: always
+    environment:
+      DASHBOARD_URL: 'https://sorry-cypress.ase.cit.tum.de'
+      EXECUTION_DRIVER: '../execution/mongo/driver'
+      MONGODB_URI: 'mongodb://sorry-cypress:sorry-cypress@mongo:27017'
+      MONGODB_DATABASE: 'sorry-cypress'
+      ALLOWED_KEYS: '<insert-cypress-key>'
+      SCREENSHOTS_DRIVER: '../screenshots/minio.driver'
+      MINIO_ACCESS_KEY: '<insert-minio-user-access-key>'
+      MINIO_SECRET_KEY: '<insert-minio-user-secret>'
```

Comment:
All these placeholders should be env vars

Reply:
Done!